molecule/default/playbook.yml

```diff
@@ -2,12 +2,11 @@
 - hosts: all
   become: true
   vars:
-    pulp_config_secret_key: secret
+    pulp_secret_key: secret
     pulp_default_admin_password: 'password'
     ansible_python_interpreter: "/usr/bin/env python"
   roles:
-    - pulp3-redis
```

Comment:
don't we still want pulp3-redis? I'm not entire sure, but we do need redis somewhere.

Reply:
The pulp3-resource-manager role declares a dependency on the pulp3-redis role.

Reply:
sweet

Reply:
that makes sense